tests/vm_networking_test.go

```diff
@@ -159,8 +165,44 @@ var _ = Describe("Networking", func() {
 			case "InboundVM":
 				addr = inboundVM.Status.Interfaces[0].IP
 			}
-			cmdCheck = fmt.Sprintf("ping %s -c 1 -w 5\n", addr)
+
+			// Check br1 MTU inside the pod
+			vmPod := tests.GetRunningPodByLabel(outboundVM.Name, v1.DomainLabel, tests.NamespaceTestDefault)
+			output, err := tests.ExecuteCommandOnPod(
+				virtClient,
+				vmPod,
+				vmPod.Spec.Containers[0].Name,
+				[]string{"ip", "address", "show", "br1"},
+			)
+			log.Log.Infof("%v", output)
```

Comment:
Is this overlooked debugging cruft?

Reply:
Not exactly. If the test passes, those are not shown in console; if the test fails, they are, and they may help to understand what happened during command execution.